packages/design-system/src/components/N8nButton/Button.vue

```diff
@@ -68,62 +56,114 @@ export default {
 			type: Boolean,
 			default: false,
 		},
-		icon: {
-		},
-		round: {
+		active: {
```

Comment:
why do we need an active prop? is this used anywhere? I thought active was more of a state when you press a button.

Reply:
Removed. Was going to add it as a separate explicit state, but we probably won't use this.